static/js/tools/scatter2/util.ts

```diff
@@ -57,26 +65,19 @@ async function getPlacesInNames(
 async function getStatsCollection(
   parent_place: string,
   child_type: string,
-  date: string,
   statVars: Array<string>
-): Promise<Record<string, SourceSeries>> {
+): Promise<Record<string, PlacePointStat>> {
   let statVarParams = "";
   for (const statVar of statVars) {
     statVarParams += `&stat_vars=${statVar}`;
   }
-  const resp = await axios.get(
-    `/api/stats/collection?parent_place=${parent_place}&child_type=${child_type}&date=${date}${statVarParams}`
-  );
-  // Tag `SourceSeries`'s with the requested date
-  const data = resp.data;
-  for (const dcid in data) {
-    const sourceSeries = data[dcid];
-    if (_.isEmpty(sourceSeries)) {
-      continue;
-    }
-    sourceSeries["date"] = date;
-  }
-  return data;
+  return axios
+    .get(
+      `/api/stats/collection?parent_place=${parent_place}&child_type=${child_type}${statVarParams}`
```

Comment:
Note this needs to be updated after https://github.com/datacommonsorg/mixer/pull/512 is submitted. Can sync mixer submodule and update this in a follow-up PR

Reply:
ok, 

> looks good!
> 
> having a separate "point" or "pointcollection" class would be helpful to organize the code. can be done in a followup

There's currently a "point" interface that's being used, how do you want me to use this new class you're suggesting?

Reply:
ok, good to know, will do that